client/src/components/Applications/AppForm/AppFormSchema.js

```diff
@@ -79,16 +68,14 @@ const FormSchema = (app) => {
     };
 
     field.type = 'text';
-    appFields.schema.fileInputs[input.name] = Yup.string();
 
-    /* TODOv3 handle fileInput validation https://jira.tacc.utexas.edu/browse/TV3-91
+    appFields.schema.fileInputs[input.name] = Yup.string();
```

Comment:
Is line 72 overwritten by 73 here?

Reply:
it isn't but would be better if combined into one statement. let me check before merging